hnn_core/tests/test_drives.py

```diff
@@ -129,9 +128,12 @@ def test_add_drives():
 
     assert net.external_drives['bursty']['n_drive_cells'] == n_drive_cells
     assert net.external_drives['bursty']['cell_specific'] == cell_specific
-    for type_name, drive in net.external_drives['bursty']['conn'].items():
-        assert drive['ampa']['A_weight'] == weights_ampa[type_name]
-        assert drive['ampa']['A_delay'] == syn_delays[type_name]
+    conn_idxs = pick_connection(net, src_gids='bursty')
```

Comment:
I can't believe it took me so long to make this function. So much easier...

Reply:
so much weekend activity. Catching up now :) 

I think the name of the function should be `pick_connections` though (plural) ... probably a separate PR however

Reply:
+1 for `pick_connections`. I'm fine with doing it here since it probably isn't too much.

Reply:
Any objections?

Reply:
let's do separate PR :) I think it would make it harder to read